includes/transformer/class-activity-object.php

```diff
@@ -25,7 +25,7 @@ public function to_object() {
 	 *
 	 * @return array The list of @-Mentions.
 	 */
-	protected function get_mentions() {
```

Comment:
Do we need to keep this around for third party plugins?

Reply:
ok, I think it is not worth changing it then! I will revert it!

Reply:
You could just deprecate it